docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -78,7 +78,8 @@ Enter the name here:
 * The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
 
 * Your final step is to push this local smaller repo back to the remote repository
-* `git push origin master`
+  * `cd ISLE.git` or `cd yourprojectnamehere-isle.git`
+  * `git push origin master`
```

Comment:
Per our slack conversation, this needs to be changed to `git push`

Reply:
Changed as agreed. Over to you for resolution and approval.